spec/index.bs

```diff
@@ -12176,6 +12176,9 @@ These operations are encoded within {{GPUComputePassEncoder}} as:
 
 Issue: describe the computing algorithm
 
+If [[WGSL#shader-execution-end|shader execution does not end]],
```

Comment:
Perhaps "in a reasonable amount of time"? (and leave "reasonable" up to the UA)

Reply:
I'd probably write:
```
If [[WGSL#shader-execution-end|shader execution end]] does not occur in a reasonable amount of time,
```
which uses the "shader execution end" wording from WGSL verbatim.